.pre-commit-config.yaml

```diff
@@ -1,13 +1,13 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 23.11.0
-    hooks:
-      - id: black
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-      - id: isort
-        args: ["--profile", "black", "--filter-files"]
+  # - repo: https://github.com/psf/black
```

Comment:
We emailed about commenting this out and acknowledged.  Note that I'll still plan to use black formatting when I create PRs.